.github/workflows/proto-linter.yml

```
@@ -0,0 +1,20 @@
+name: Layotto Env Pipeline 🌊
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

```suggestion
    name: "🍀 Proto Validation"
    runs-on: ubuntu-latest
```
We can add a name for it :)